app/lib/importer/statuses_index_importer.rb

```diff
@@ -77,4 +84,8 @@ def local_votes_scope
   def local_statuses_scope
     Status.local.select('"statuses"."id", COALESCE("statuses"."reblog_of_id", "statuses"."id") AS status_id')
   end
+
+  def local_replied_scope
+    Status.local.where.not(in_reply_to_id: nil).where('in_reply_to_account_id != account_id').select(:id, :in_reply_to_id)
```

Comment:
Maybe with this, you wouldn't need the `get_status_id` method?
```suggestion
    Status.local.where.not(in_reply_to_id: nil).where('in_reply_to_account_id != account_id').select(:id, 'in_reply_to_id AS status_id')
```